presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -0,0 +1,372 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativeworker;
+
+import com.facebook.presto.common.type.BigintType;
+import com.facebook.presto.common.type.BooleanType;
+import com.facebook.presto.common.type.DoubleType;
+import com.facebook.presto.common.type.Type;
+import com.facebook.presto.common.type.VarcharType;
+import com.facebook.presto.testing.MaterializedResult;
+import com.facebook.presto.testing.MaterializedRow;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.testng.Assert.fail;
+
+public class ContainerQueryRunnerUtils
+{
+    private ContainerQueryRunnerUtils() {}
+
+    static Path createTpchProperties()
+    {
+        try {
+            Path tempTpchProperties = Files.createTempFile("tpch", ".properties");
+            String properties =
+                    "connector.name=tpch\n" +
+                            "tpch.column-naming=STANDARD"
+                            + System.lineSeparator();
+            Files.write(tempTpchProperties, properties.getBytes(), StandardOpenOption.APPEND);
+            return tempTpchProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create tpch.properties and was exited with error message: " + e.getMessage());
```

Comment:
fail isn't really a good match here and below. The setup failed, not the test. Drop the try catch and simply let the exception bubble up into the test runner.

Reply:
Update the code.